roq-core/src/main/java/org/roqmessaging/core/ShutDownMonitor.java

```diff
@@ -63,7 +63,7 @@ public void run() {
 
 		// 2. Start the main run of the monitor
 		while (this.active) {
-			items.poll(80);
+			items.poll(100);
```

Comment:
Did you change the zmq polling time for a specific reason? Were there issues with 80?